routes/schedule.route.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const httpStatus = require('http-status');
+const userRoutes = require('./user.route');
+const authRoutes = require('./auth.route');
+const sendResponse = require('../helpers/response');
+const scheduleModel = require('../models/schedule.model');
+const modelCtrl = require('../controllers/schedule.controller');
+
+const router = express.Router();
+
+router.post('/', (req, res) => {
```

Comment:
same as this, it should be a functionality in the `schedule.controller.js` file
```suggestion
router.post('/', modelCtrl.createSchedule)
```